dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -146,23 +146,22 @@ boolean isSystemClass(String className) {
   }
 
   /**
-   * Scans class files in the jar file and returns a {@link SymbolReferenceSet} populated with
-   * symbol references.
-   *
-   * @param jar absolute path to a jar file
+   * Returns a set of symbol references (classes to class symbols, method symbols and field symbols)
```

Comment:
Is this a set? sounds more like a map.

Reply:
Yes, a symbol reference is a map entry from a class to a symbol. Updated description.